installer/release.sh

```diff
@@ -218,6 +221,23 @@ test -z "$GITCONFIG_PATH" || {
 	LIST="$(echo "$LIST" | grep -v "^$GITCONFIG_PATH\$")"
 }
 
+# ARM64 Windows handling
+IS_ARM64=
+if test -n "$arm64_artifacts_directory"
+then
+	echo "Including ARM64 artifacts from $arm64_artifacts_directory";
+	IS_ARM64=1
+	# If directory starts with a slash (i.e. /arm64), strip it, because InnoSetup can't handle it.
+	if [[ $arm64_artifacts_directory == *\/* ]];
```

Comment:
Do you really want to catch paths starting with a `/`? If so, let's do this instead:

```sh
case "$arm64_artifacts_directory" in
/*)
    arm64_artifacts_directory=${arm64_artifacts_directory#/}
    ;;
esac
```

But since the `${...#<prefix>}` construct will strip the given prefix only when found, that `case` is unnecessary. Even better: we do not need to modify the variable, but use that construct instead below.